docker/tests/scripts/test-fedora-binaries.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2023 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+set -euo pipefail
+
+for version in $(jq -r '.fedora[]' docker/supported_versions.json); do
+    docker build --build-arg dist="$version" --build-arg repo="Tezos" -t fedora-test -f docker/tests/Dockerfile-fedora-test .
```

Comment:
Could you please clarify why here `Tezos` arg is capitalized, but in Ubuntu test it isn't? 🤔

Reply:
Because, unfortunately, on fedora and on ubuntu packages are named differently, see here https://serokell.slack.com/archives/C013LJZVC1K/p1690898759870919?thread_ts=1690819686.455979&cid=C013LJZVC1K